lib/js-api/src/api/content-rest-api/docs/NodesApi.md

```diff
@@ -25,6 +25,8 @@ All URIs are relative to *https://localhost/alfresco/api/-default-/public/alfres
 | [unlockNode](#unlockNode)                                           | **POST** /nodes/{nodeId}/unlock                         | Unlock a node                                         |
 | [updateNode](#updateNode)                                           | **PUT** /nodes/{nodeId}                                 | Update a node                                         |
 | [updateNodeContent](#updateNodeContent)                             | **PUT** /nodes/{nodeId}/content                         | Update node content                                   |
+| [initialFolderSizeCalculation](#initialFolderSizeCalculation)       | **POST** /nodes/{nodeId}/size-details                   | Initiate a new request to calculate folder size       |
```

Comment:
I think typo, actually that function is named: initiateFolderSizeCalculation, not initialFolderSizeCalculation. Please rename it in documentation

Reply:
Yes, it was a typo. Fixed the issue now